menas/pom.xml

```diff
@@ -251,7 +251,7 @@
         <dependency>
             <groupId>org.webjars.bower</groupId>
             <artifactId>chart.js</artifactId>
-            <version>2.7.3</version>
+            <version>${bower.chart.js.version}</version>
```

Comment:
I don't think this is needed, because it will be replaced in the separation PR

Reply:
I see, that it will be probably the case.
But just to be sure, I would keep this one now, and removed when not needed.